learn-more.css

```diff
@@ -1,350 +0,0 @@
-        url(side-overlay.png);
-    background-repeat: no-repeat;
-        
-
-}
-/* button CSS below------------------------ */
-
-.date-label {
-    background-color: #764248;
-    border-radius: 6px;
-    color: #FDE6BE;
-    padding: .25rem;
-    width: 25%;
-    height: auto;
-    box-shadow: darkgrey;
-    font-family: "Raleway", light;
-}
-
-.date-label:hover {
-    background-color: #61373B;
-}
-
-.styled-btn {
-    background-color: #8F4307;
-    color: #FDE6BE;
-    font-size: 1rem;
-    border: 0;
-    padding: 0 20px;
-    text-align: center;
-    border-radius: 5px;
-    line-height: 2.5;
-}
-
-.styled-btn:hover {
-    background-color: rgb(161, 83, 30);
-}
-
-/* noteworthy area CSS below------------------------------ */
-
-.notables {
-    display: flex;
-    flex-direction: column;
-    justify-content: space-around;
-}
-
-.notables h3 {
-    color: #4B3021;
-    font-family: "Open Sans Condensed";
-    text-align: center;
-    text-transform: uppercase;
-}
-
-.notables p {
-    color: #4B3021;
-    font-family: 'Raleway', light;
-    text-align: justify;
-}
-
-.notables img {
-    float: left;
-    height: 7rem;
-    width: 7rem;
-    padding-right: 15px;
-   padding-top: 15px;
-   padding-top: 10px;
-}
-
-/* footer CSS Below ---------------------------------*/
-#footer {
-    margin: 1em 0 0 0;
-    padding: 1em;
-    background-color: rgb(145, 162, 168);
-    border-top: 1px solid rgb(122, 117, 117);
-    box-shadow: 0 50vh 0 50vh rgb(145, 162, 168);
-}
-h2#footer-title {
-    text-align: center;
-    margin: 5vh auto 3vh auto;
-    font-size: 1em;
-    font-weight: lighter;
-    letter-spacing: 5px;
-    }
-div#form-space {
-        display: flex;
-        justify-content: space-evenly;
-      }
-
-div#form {
-        display: grid;
-        grid-template-columns: 1fr 1fr;
-        grid-template-rows: 1fr 3fr 1fr;
-        grid-gap: 0.5em;
-        grid-template-areas:
-          "  customer-name      customer-email"
-          "  customer-message   customer-message"
-          "  send-button        . ";
-        margin: 1em;
-        width: 42.5vw;
-      }
-      
-#customer-name {
-        grid-area: customer-name;
-        font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-        font-size: 10px;
-        padding-left: 10px;
-      }
-
-#customer-name:focus {
-        background-color: grey;
-    }
-#customer-email {
-        grid-area: customer-email;
-        font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-        font-size: 10px;
-        padding-left: 10px;
-      }
-
-#customer-email:focus {
-        background-color: grey;
-    } 
-#customer-message {
-    grid-area: customer-message;
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-    font-size: 10px;
-    padding-left: 10px;
-    padding-top: 10px;
-  }
-#customer-message:focus{
-    background-color: grey;
-}
-#contact-info{
-    display:grid;   
-    grid-template-rows: repeat(7, 1fr);
```

Comment:
An excellent way to make a bunch of rows of the same size. Or you could just not specify rows, and allow your container to use as many rows as it needs.